libraries/botbuilder-schema/botbuilder/schema/teams/_models_py3.py

```diff
@@ -2058,3 +2086,265 @@ def __init__(
         self.user = user
         self.meeting = meeting
         self.conversation = conversation
+
+
+class TabContext(Model):
+    """
+    Current tab request context, i.e., the current theme.
+
+    :param theme: Gets or sets the current user's theme.
+    :type theme: str
+    """
+
+    _attribute_map = {
+        "theme": {"key": "theme", "type": "str"},
+    }
+
+    def __init__(self, *, theme=None, **kwargs) -> None:
+        super(TabContext, self).__init__(**kwargs)
+        self.theme = theme
+        self._custom_init()
+
+    def _custom_init(self):
+        return
+
+
+class TabRequest(Model):
+    """
+    Invoke ('tab/fetch') request value payload.
+
+    :param tab_entity_context: Gets or sets current tab entity request context.
+    :type tab_entity_context:
+     ~botframework.connector.teams.models.TabEntityContext
+    :param context: Gets or sets current tab entity request context.
+    :type context:
+     ~botframework.connector.teams.models.TabContext
+    :param state: Gets or sets state, which is the magic code for OAuth Flow.
+    :type state: str
+    """
+
+    _attribute_map = {
+        "tab_entity_context": {"key": "tabContext", "type": "TabEntityContext"},
+        "context": {"key": "context", "type": "TabContext"},
+        "state": {"key": "state", "type": "str"},
+    }
+
+    def __init__(
+        self, *, tab_entity_context=None, context=None, state=None, **kwargs
+    ) -> None:
+        super(TabRequest, self).__init__(**kwargs)
+        self.tab_entity_context = tab_entity_context
+        self.context = context
+        self.state = state
+        self._custom_init()
+
+    def _custom_init(self):
+        return
+
+
+class TabResponseCard(Model):
+    """
+    Envelope for cards for a Tab request.
+
+    :param card: Gets or sets adaptive card for this card tab response.
+    :type card: object
+    """
+
+    _attribute_map = {
+        "card": {"key": "card", "type": "object"},
+    }
+
+    def __init__(self, *, card=None, **kwargs) -> None:
+        super(TabResponseCard, self).__init__(**kwargs)
+        self.card = card
+        self._custom_init()
+
+    def _custom_init(self):
+        return
+
+
+class TabResponseCards(Model):
+    """
+    Envelope for cards for a TabResponse.
+
+    :param cards: Gets or sets adaptive card for this card tab response.
+    :type cards: list[object]
+    """
+
+    _attribute_map = {
+        "cards": {"key": "cards", "type": "[object]"},
```

Comment:
@axelsrz Are you able to change the type to something like `[TabResponseCard]` or whatever the Python equivalent is of [this](https://github.com/microsoft/botbuilder-js/pull/3224/files#diff-348dcf11b9d343d90e766394f8ebae3c3675bde51e95705520bd15451f0c5720R1308) (an array of `TabResponseCard`).

@EricDahlvang Do we still need this change in .NET?

Reply:
@mdrichardson this if fixed in dotnet in this PR: https://github.com/microsoft/botbuilder-dotnet/pull/5185/files#diff-5a0a768d82736568f8e996abff4ff3eacea0696d1ed47f928822bfc2d6f895d2R30